myhomework2.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+
+// не звертайте на цю функцію уваги
+// вона потрібна для того щоб правильно зчитати вхідні дані
+function readHttpLikeInput()
+{
+    $f = fopen('php://stdin', 'r');
+    $store = "";
+    $toread = 0;
+    while ($line = fgets($f)) {
```

Comment:
place empty lines before and after code blocks (if, for, while, do while etc.)
the code will be more readable and understandable in such a way
also, before return, break, continue and so on

Reply:
I understood your advice, but this function was written for me, it was given with the task

Reply:
it doesn't matter. Your code should has same style within the whole project. Otherwise, it will look strange. 
Imagine that you get the project where in every file code is written differently. Will it be fine?